selfservice/hook/web_hook.go

```diff
@@ -352,6 +353,15 @@ func (e *WebHook) execute(ctx context.Context, data *templateContext) error {
 
 		resp, err := httpClient.Do(req)
 		if err != nil {
+			if te := interface{ Timeout() bool }(nil); errors.As(err, &te) && te.Timeout() || errors.Is(err, context.DeadlineExceeded) {
```

Comment:
The condition could be a function with a nice name to make the code more readable
```go
func isTimeoutError(err error) boolean {
	te := interface{ Timeout() bool }(nil)
	return errors.As(err, &te) && te.Timeout() || errors.Is(err, context.DeadlineExceeded)
}
```

```suggestion
			if isTimeoutError(err) {
```

Reply:
Yep, that's fair. Will fix.